src/pages/playerUrls.js

```diff
@@ -741,4 +741,8 @@ module.exports = {
   doflix: {
     match: ['*://doflix.net/*'],
   },
+  // anilib
+  anilib: {
```

Comment:
I did not find any player that would need this. Do you have an example. If the player is directly on the page without an iframe then no additional injection is necesseary.

Reply:
Was long time since then, was thought that any new player should be added manually, fixed it